_clients/data-prepper/data-prepper-reference.md

```diff
@@ -176,17 +173,10 @@ Prior to Data Prepper 1.3, Processors were named Preppers. Starting in Data Prep
 {: .note }
 
 
-### otel_trace_raw_prepper
-
-Converts OpenTelemetry data to OpenSearch-compatible JSON documents and fills in trace group related fields in those JSON documents. It requires `record_type` to be set as `otlp` in `otel_trace_source`.
-
-Option | Required | Type | Description
-:--- | :--- | :--- | :---
-trace_flush_interval | No | Integer | Represents the time interval in seconds to flush all the descendant spans without any root span. Default is 180.
-
 ### otel_trace_raw
 
-This processor is a Data Prepper event record type compatible version of `otel_trace_raw_prepper` that fills in trace group related fields into all incoming Data Prepper span records. It requires `record_type` to be set as `event` in `otel_trace_source`.
+This processor is a Data Prepper event record type replacement of `otel_trace_raw_prepper` (no longer supported since Data Prepper 2.0). 
+The processor fills in trace group related fields into all incoming Data Prepper span records by statefully caching the root span info per traceId. 
```

Comment:
"...fills in trace group related fields **in** all incoming..."
"state caching" appears to be acceptable usage. OK here?
Also, Is it appropriate to spell out "info"?